src/RemoteStorage/Provider.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+
+const Context = React.createContext({});
+
+export const useRemoteStorageApi = () => React.useContext(Context);
+
+const Provider = ({ resources, mutator, ...rest }) => {
+  const [persistentMutator] = React.useState(mutator);
+
+  React.useEffect(() => {
```

Comment:
why do you access react utils via React? looks like it's done only by you

Reply:
Kent C. Dodds taught me bad
https://kentcdodds.com/blog/useeffect-vs-uselayouteffect

Reply:
that's why you don't follow project codestyle?)